src/Azure/Orleans.Streaming.EventHubs/Providers/EventDataGeneratorStreamProvider/EventHubPartitionDataGenerator.cs

```diff
@@ -74,24 +79,14 @@ public bool TryReadEvents(int maxCount, out IEnumerable<EventData> events)
 
         private static IEnumerable<int> GenerateEvent(int sequenceNumber)
         {
-            var events = new List<int>();
-            events.Add(sequenceNumber);
-            return events;
+            return [sequenceNumber];
         }
 
         public static Func<StreamId, IStreamDataGenerator<EventData>> CreateFactory(IServiceProvider services)
         {
             return (streamId) => ActivatorUtilities.CreateInstance<SimpleStreamEventDataGenerator>(services, streamId);
         }
 
-
-        private class WrappedEventData : EventData
-        {
-            public WrappedEventData(ReadOnlyMemory<byte> eventBody, IDictionary<string, object> properties = null, IReadOnlyDictionary<string, object> systemProperties = null, long sequenceNumber = long.MinValue, long offset = long.MinValue, DateTimeOffset enqueuedTime = default, string partitionKey = null) : base(eventBody, properties, systemProperties, sequenceNumber, offset, enqueuedTime, partitionKey)
```

Comment:
This type is based on a deprecated constructor that we've kept for back-compat, but do not recommend using.   The `EventHubModelFactory` is the right approach to create data that sets broker-owned fields.